lib/android-helpers.js

```diff
@@ -467,11 +467,11 @@ helpers.initDevice = async function (adb, opts) {
   }
   if (!opts.avd) {
```

Comment:
we seen `opts.avd` fail with @imurchie,  since you can avoid setting the avd capability if the emulator is already running